.github/workflows/ci.yml

```
@@ -27,6 +27,8 @@ jobs:
     - name: Run
       run: |
         make run
+      env:
+          COVALENT_API_KEY: ${{ secrets.COVALENT_KEY }}
```

```suggestion
          COVALENT_API_KEY: ${{ secrets.COVALENT_API_KEY }}
```

Can we name it the same?